requirements/edx/github.txt

```diff
@@ -46,6 +46,7 @@
 # Third-party:
 -e git+https://github.com/cyberdelia/django-pipeline.git@1.5.3#egg=django-pipeline
 git+https://github.com/edx/django-wiki.git@v0.0.5#egg=django-wiki==0.0.5
+git+https://github.com/edx/django-oauth2-provider.git@70de1717952359d606c92b6b48aef1d0fefa84cb#egg=django-oauth2-provider==0.6.0a1
```

Comment:
Is this yet-another DOP library?  Different from `edx-django-oauth2-provider`, which is installed in base.txt?